src/main/java/access/Application.java

```diff
@@ -55,10 +100,6 @@
 	private int threadCountSize;
 	@Value("${thread.count.limit}")
 	private int threadCountLimit;
-	@Value("${http.max.total}")
```

Comment:
I would leave these at the class level, still; and use them in the basic profile as we did before.